packages/flame/lib/src/sprite_batch.dart

```diff
@@ -67,7 +67,7 @@ class BatchItem {
     required this.source,
     required this.transform,
     required this.color,
-  })   : matrix = Matrix4(
+  })  : matrix = Matrix4(
```

Comment:
Still one ` ` too much?

Reply:
you have no idea how long it took me to find this. locally it didn't complain

Reply:
I still se the extra empty space here